test/basic.test.js

```diff
@@ -28,19 +28,19 @@ describe('test based on examples', () => {
     const ironShovelItem = new Item(472, 1)
 
     it('constructor makes item correctly', () => {
-      const expectedObj = { count: 1, displayName: 'Iron Shovel', metadata: 0, name: 'iron_shovel', nbt: null, stackSize: 1, type: 472 }
+      const expectedObj = { count: 1, displayName: 'Iron Shovel', metadata: 0, name: 'iron_shovel', nbt: { name: '', type: 'compound', value: { Damage: { type: 'int', value: 0 } } }, stackSize: 1, type: 472, stackId: 0 }
```

Comment:
Why the change with nbt?

Reply:
As in vanilla as far as I have tested on both java and bedrock, any item which can lose durability has the Damage tag set to 0 by default, so I added that to the constructor; then I had to change it here as the iron shovel item will have the Damage tag in NBT